libwallet/assets/dcr/dex_wallet.go

```diff
@@ -69,26 +78,26 @@ func (dw *DEXWallet) SpvMode() bool {
 // Accounts returns the names of the accounts for use by the exchange wallet.
 func (dw *DEXWallet) Accounts() dexdcr.XCWalletAccounts {
 	var accts dexdcr.XCWalletAccounts
-	accountName, err := dw.AccountName(dw.tradingAccountNumber)
+	accountName, err := dw.w.AccountName(context.Background(), uint32(dw.tradingAccountNumber))
```

Comment:
This or to add `AccountName(acctNumber)` to walletHelper.

Reply:
This is fine. We can also save the ctx from `dw.Connect` and use it here and anywhere else.